engine/src/main/java/org/terasology/rendering/openvrprovider/OpenVRState.java

```diff
@@ -38,6 +38,7 @@
     // In the head frame
     private Matrix4f[] eyePoses = new Matrix4f[2];
     private Matrix4f[] projectionMatrices = new Matrix4f[2];
+    private float groundPlaneYOffset = 0.0f;
```

Comment:
Is this the same as the height of the character's eyes?  If it is, probably the more resilient way to get this value would be from the character that the player is controlling.  However,  I suppose that making this different from the character is good so that VR nausea is avoided?